app/App.tsx

```diff
@@ -8,7 +8,7 @@ import Navigation from './components/ui/Navigation/Navigation';
 // import CategoryPage from './find/FindPage';
 // import ResourcesTable from './search/ResourcesTable';
 import { round } from './utils/index';
-import { getSiteTitle, getSiteUrl, isSFFamiliesSite } from './utils/whitelabel';
+import configurations from './utils/whitelabel';
```

Comment:
Could you name default imports the same as the file being imported, e.g. name this `whitelabel`, not `configurations? That will reduce cognitive load by not introducing a new name. If you don't think the name `whitelabel` is good, then we may want to rename that file.